frontend/device/pocketbook/device.lua

```diff
@@ -402,12 +402,13 @@ elseif codename == "PocketBook 611" then
     return PocketBook611
 elseif codename == "PocketBook 613" then
     return PocketBook613
-elseif codename == "PocketBook 614W" or codename == "PocketBook 614" then
+elseif codename == "PocketBook 614" or codename == "PocketBook 614W" then
     return PocketBook614W
-elseif codename == "PocketBook 615" or codename == "PB615" then
+elseif codename == "PB615" or codename == "PB615W" then
+    codename == "PocketBook 615" or codename == "PocketBook 615W" or
```

Comment:
Typo for this I imagine:
```suggestion
elseif codename == "PB615" or codename == "PB615W" or
    codename == "PocketBook 615" or codename == "PocketBook 615W" then
```